index.bs

```diff
@@ -126,7 +126,7 @@ Message = (
 )
 
 CommandResponse = (
-  id: uint,
+  id: uint?,
```

Comment:
A command response message, by definition, must always have an ID. Can the definitions be split up / reworked so that command responses always have a valid ID and errors have an optional ID? The meaning may end up being the same, but I think this will ultimately be more clear to readers.

Reply:
Also, I think this should be `uint / null`.

Reply:
Makes sense, fixed.